wrangler-core/src/main/java/io/cdap/wrangler/utils/RecordConvertor.java

```diff
@@ -91,6 +92,8 @@ public StructuredRecord decodeRecord(Row row, Schema schema) throws RecordConver
           builder.setTimestamp(name, (ZonedDateTime) decodedObj);
         } else if (decodedObj instanceof BigDecimal) {
           builder.setDecimal(name, (BigDecimal) decodedObj);
+        } else if (decodedObj instanceof LocalDateTime) {
```

Comment:
is there corresponding logic somewhere when converting StructuredRecord to a Row to create a LocalDateTime instead of the underlying string?

Reply:
wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java getValue() method returns LocalDateTime to set to Row (In this PR). Applicable directives will also be setting LocalDateTime to Row.